roles/pul_nomad/files/pul_nomad_policy.hcl

```diff
@@ -3,9 +3,12 @@ agent_prefix "" {
 }
 
 node_prefix "" {
-  policy = "read"
```

Comment:
what about imagecat necessitated this change? would we have needed it eventually for dpul-c but we just didn't bump up against it yet?

Reply:
Yeah. The nomad logs had some errors in it I only noticed because I was in there poking around. It could be a separate PR.

Reply:
A separate PR would be nice